Assets/MixedRealityToolkit/Utilities/StandardShader/StandardShaderUtility.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Utilities
```

Comment:
Also I wonder if this should go into SDK, not core MRTK

Reply:
Since the shader and it's utilities live in core, I believe this also needs to stay in core since it's a dependency.